CSS/CSS.sublime-syntax

```diff
@@ -725,6 +727,11 @@ contexts:
             1: punctuation.definition.entity.css
         - match: \*
           scope: entity.name.tag.wildcard.css
+          # Combinators
+          # https://drafts.csswg.org/selectors-4/#combinators
+          # https://drafts.csswg.org/css-scoping/#deep-combinator
+        - match: '({{combinators}})(?!{{combinators}})'
```

Comment:
Won't that just result in the first `+` being unscoped? Perhaps instead we should match any further combinators and mark them as illegal?